style.css

```diff
@@ -750,3 +756,370 @@ ul {
   margin-bottom: 20px;
   color: #172b4d;
 }
+
+/* Desktop version */
+@media screen and (min-width: 768px) {
+  body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    overflow-x: hidden;
+  }
+
+  .logo {
+    font-size: 25px;
+    font-weight: 700;
+    line-height: 20px;
+    background-color: transparent;
+  }
+
+  .app-header {
+    display: flex;
+  }
+
+  .humbourger,
+  .ambony {
+    display: none;
+  }
+
+  .menu {
+    display: flex !important;
+    justify-content: flex-end;
+    align-items: center;
+    margin-right: 20px;
+    gap: 15%;
+  }
+
+  .menu-item a {
+    display: flex;
+    color: #344563;
+    text-decoration: none;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 20px;
+    text-align: center;
+  }
+
+  .header-ambony {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    width: 100%;
+    padding-left: 10%;
+    background-color: #fff;
+    background-image: url(media/Header\ bg.svg);
+    background-size: cover;
+    top: 2.875rem;
+    border-bottom-left-radius: 60px;
+    height: 44.88rem;
+  }
+
+  .header-container {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    padding-left: 10%;
+  }
+
+  .narson {
+    width: 100%;
+  }
+
+  .detail {
+    height: 98px;
+    width: 920px;
+    left: 0;
+    top: 134px;
+  }
+
+  .connection {
+    margin-top: -1px;
+    font-size: 16px;
+    font-weight: 500;
+    line-height: 24px;
+    text-align: left;
+    color: #7f8cff;
+  }
+
+  /* work section */
+
+  .work {
+    display: grid;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 86.5%;
+    margin: 10px 0 0 28px;
+    height: auto;
+  }
```

Comment:
- [ ] Ouch! there seems to be a gray background in the works section which isn't found in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=Vim0xz58yI4QkbS9-0) so please make sure you stick to the given design as this ensures your future clients you can properly implement a given design down to the last pixel. ✔
    - Your design:
      ![image](https://user-images.githubusercontent.com/104968236/230616209-a4a0938b-6cb9-442f-ab80-29760902e50c.png)
    - Figma design:
      ![image](https://user-images.githubusercontent.com/104968236/230616413-b1077963-c358-4dd9-8009-5ee5b60d9b31.png)
     - As you can see the background color of the works section is `#f4f5f7` and not gray so kindly fix that. 💪